.ado/windows-vs-pr.yml

```diff
@@ -315,6 +315,49 @@ jobs:
           #rerunFailedTestCasesMaxLimit: '5' # Optional
           #rerunMaxAttempts: '3' # Optional
 
+      - task: VSTest@2
+        displayName: Run Desktop Unit Tests (Google Test)
+        timeoutInMinutes: 5 # Set smaller timeout for UTs, since there have been some hangs, and this allows the job to timeout quicker
+        inputs:
+          testSelector: 'testAssemblies' # Options: testAssemblies, testPlan, testRun
+          testAssemblyVer2: 'JSI.Desktop.UnitTests\JSI.Desktop.UnitTests.exe'  # Required when testSelector == TestAssemblies
+          #testPlan: # Required when testSelector == TestPlan
+          #testSuite: # Required when testSelector == TestPlan
+          #testConfiguration: # Required when testSelector == TestPlan
+          #tcmTestRun: '$(test.RunId)' # Optional
+          searchFolder: '$(Build.SourcesDirectory)\vnext\target\$(BuildPlatform)\$(BuildConfiguration)'
+          #testFiltercriteria: # Optional
+          #runOnlyImpactedTests: False # Optional
+          #runAllTestsAfterXBuilds: '50' # Optional
+          #uiTests: false # Optional
+          #vstestLocationMethod: 'version' # Optional. Options: version, location
+          vsTestVersion: '15.0' # Optional. Options: latest, 15.0, 14.0, toolsInstaller
+          #vstestLocation: # Optional
+          #runSettingsFile: # Optional
+          #overrideTestrunParameters: # Optional
+          pathtoCustomTestAdapters: '$(Build.SourcesDirectory)\vnext\packages\GoogleTestAdapter.0.16.1\build\_common' # Optional
+          #runInParallel: False # Optional
+          runTestsInIsolation: true # Optional
+          #codeCoverageEnabled: False # Optional
+          #otherConsoleOptions: # Optional
+          #distributionBatchType: 'basedOnTestCases' # Optional. Options: basedOnTestCases, basedOnExecutionTime, basedOnAssembly
+          #batchingBasedOnAgentsOption: 'autoBatchSize' # Optional. Options: autoBatchSize, customBatchSize
+          #customBatchSizeValue: '10' # Required when distributionBatchType == BasedOnTestCases && BatchingBasedOnAgentsOption == CustomBatchSize
+          #batchingBasedOnExecutionTimeOption: 'autoBatchSize' # Optional. Options: autoBatchSize, customTimeBatchSize
+          #customRunTimePerBatchValue: '60' # Required when distributionBatchType == BasedOnExecutionTime && BatchingBasedOnExecutionTimeOption == CustomTimeBatchSize
+          #dontDistribute: False # Optional
+          #testRunTitle: # Optional
+          platform: $(BuildPlatform) # Optional
+          configuration: $(BuildConfiguration) # Optional
+          publishRunAttachments: true # Optional
+          #diagnosticsEnabled: True # Optional
+          collectDumpOn: 'onAbortOnly' # Optional. Options: onAbortOnly, always, never
```

Comment:
>onAbortOnly [](start = 26, length = 11)

(nit) what about timeout?

Reply:
The [documentation](https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/test/vstest?view=azure-devops) for running a Visual Studio Test Test on Azure DevOps does not list any option here that explicitly mentions timeouts, so this is the best that I can do for now.